sdk/ml/azure-ai-ml/azure/ai/ml/operations/_feature_set_operations.py

```diff
@@ -159,6 +159,10 @@ def begin_create_or_update(self, featureset: FeatureSet, **kwargs: Dict) -> LROP
         featureset_copy.properties["featuresetProperties"] = json.dumps(featureset_spec._to_dict())
 
         sas_uri = None
+
+        with open(os.path.join(featureset_copy.path, ".amlignore"), mode="w", encoding="utf-8") as f:
```

Comment:
Just to confirm, is this `featureset_copy.path` a temp path? And not the path where the featuresetspec.dump wrote the files?

Reply:
no, it's the feature set spec path, I've verified that it will create a .amlignore file in feature set spec folder.

Reply:
Got it.